python/tests/util/mark.py

```diff
@@ -5,11 +5,45 @@
 
 As of the Change Date specified in that file, in accordance with the Business Source License, use of this software will be governed by the Apache License, version 2.0.
 """
-
+import os
+import sys
+import pytest
 from typing import Union
 from datetime import date
 from numpy import datetime64
 
+# TODO: Some tests are either segfaulting or failing on MacOS with conda builds.
+# This is meant to be used as a temporary flag to skip/xfail those tests.
+MACOS_CONDA_BUILD = sys.platform == "darwin" and os.getenv("ARCTICDB_USING_CONDA", "0") == "1"
+_MACOS_CONDA_BUILD_SKIP_REASON = (
+    "Tests fail for macOS conda builds, either because Azurite is improperly configured"
+    "on the CI or because there's problem with Azure SDK for C++ in this configuration."
+)
+
+# These two should become pytest marks as opposed to variables feeding into skipif
+PERSISTENT_STORAGE_TESTS_ENABLED = os.getenv("ARCTICDB_PERSISTENT_STORAGE_TESTS") == "1"
+FAST_TESTS_ONLY = os.getenv("ARCTICDB_FAST_TESTS_ONLY") == "1"
```

Comment:
Thanks for keeping `FAST_TESTS_ONLY` alive, I added it recently just for local development, otherwise the feedback cycle is horrific.